### PR TITLE
allows for a value instead of an array in life-event.md

### PR DESCRIPTION
**Pull request by fpigeonjr**

### CODE REVIEW

File: content/readme.md [hunk 1]
```diff
@@ -1,19 +1,48 @@
...
-  - You are also able to put in multiple dates / encodings to create a range∂
-  - i.e. You wish to only have a user be elgible if they are older than 40 years and less than 60 years
-  - acceptanceValues: ["<60years", ">40years"]
+## Top-Level Filters
+
+In order to provide BEARS users with a reduced and tailored list of eligibility questions you can use the following encoding in a `life-event.md` file:
+
+```yaml
+// military-service.md
+
```
Comment (jschoenemann-cms) [inline content/readme.md:9]: prob didnt mean to keep this one here.

File: components/SummaryBox.vue [hunk 2]
```diff
@@ -7,28 +7,24 @@
...
-              </div>
+            <template v-for="filter in topLevelFilters">
+              <h3
+                id="summary-box-key-information"
+                :key="`heading-${filter.criteriaKey}`"
+                class="usa-summary-box__heading">
+                {{ filter.label }}
+              </h3>
+              <template v-for="criterion in getCriteriaMap(filter.criteriaKey)">
+                <CriteriaChild
```
Comment (jschoenemann-cms) [inline components/SummaryBox.vue:18]: we should still look for getting this to load the one object instead of looping through an array of one value

**Review by jschoenemann-cms — CHANGES_REQUESTED** [review]

File: content/readme.md (hunk 1 above)
Comment (fpigeonjr) [reply content/readme.md:9]: I thought it may be a good idea to reference the template in this PR.

File: test/Components/SummaryBox.spec.js [hunk 3]
```diff
@@ -32,9 +31,9 @@ describe("<SummaryBox />", () => {
   beforeEach(() => {
     criteriaState.namespaced = true
 
-    actions = {
-      getCriteriaMap: jest.fn(),
-    }
+    // actions = {
```
Comment (fpigeonjr) [inline test/Components/SummaryBox.spec.js:34]: we can probably remove these comments

**Review by lgarceau768 — APPROVED** [review]
Looks good and works on my local as well!

**Review by r-bartlett-gsa — APPROVED** [review]
Looks good!

**Review by jschoenemann-cms — APPROVED** [review]
Looks good.

**Pull request merged**